src/test/functional/functional-tests.js

```diff
@@ -1085,9 +1105,14 @@ describe('functional tests', function() {
  
       // This test is very similar to that above, except it does not include
       // Minio.ObjectCreatedAll in the config. Thus, no events should be emitted.
-      step('listenBucketNotification(bucketName, prefix, suffix, events)_events:ObjectRemoved_', done => {
+      step(`listenBucketNotification(bucketName, prefix, suffix, events)_bucketName:${bucketName}, events:s3:ObjectRemoved:*`, done => {
```

Comment:
Does making minio-js functional test work with GCS requires all of this cleanup? @kannappanr can we do it in another PR?